tox.ini

```diff
@@ -33,6 +33,12 @@ commands =
     poetry run coverage html
     open ./htmlcov/index.html
 
+[testenv:no_external]
+commands =
+    poetry run pytest --doctest-modules singer_sdk
+    poetry run pytest tests/core
+    poetry run pytest tests/cookiecutters
+
```

Comment:
We should probably use [pytest markers](https://docs.pytest.org/en/stable/example/markers.html#marking-individual-tests-when-using-parametrize) to (de)select external tests.